src/noggit/World.cpp

```diff
@@ -2124,6 +2127,67 @@ void World::selectVertices(math::vector_3d const& pos, float radius)
   });
 }
 
+void World::delete_models(std::vector<selection_type>& const types)
+{
+   _model_instance_storage.delete_instances(types);
+  need_model_updates = true;
+}
+
+void World::selectVertices(math::vector_3d const& pos1, math::vector_3d const& pos2)
+{
+  math::vector_3d pos_min = math::vector_3d(std::min(pos1.x,pos2.x),std::min(pos1.y,pos2.y),std::min(pos1.z,pos2.z));
+  math::vector_3d pos_max = math::vector_3d(std::max(pos1.x,pos2.x),std::max(pos1.y,pos2.y),std::max(pos1.z,pos2.z));
+  _vertex_center_updated = false;
```

Comment:
`select_all_chunks_between` selects chunks for the script selections, while this version of selectVertices is just a square version of the other selectVertices, so I think that's intended since the other selectVertices only sets them to false too (`updateVertexCenter`/`vertexBorderChunks` sets them to true, I think that would apply to this one as well).

I can see however that I never wrapped this function, I probably wrote it before I had sketched out our own selection mechanism.